test/hotspot/jtreg/compiler/c2/Test8183543.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Place test into `compiler/regalloc`.
We are avoiding using bug Id in test name now. Use name which explains what test does.